### ISSUE TITLE
Mapnik rendering error - left top corner

### ISSUE THREAD

**Issue by mar0321** [mapnik/mapnik#3820]
Hi,
im using nik4.py with mapnik to render raster images. Everything is fine expect one thing. In left top corner there is a tiny part which is rendered in more intense colour so in map(all rasters) e.g in forest there are green triangles on the border of rasters, how can i eliminate this thing ? Or what can i/should i do ?

Thank u all for ideas.

**Comment by talaj** [comment]
Hi,
do you use blur? If yes, try to set [`image-filters-inflate="true"`](http://mapnik.org/mapnik-reference/#3.0.6/image-filters-inflate) to the Style node.

**Comment by mar0321** [comment]
Thanks i'll try, but im' not using blur, here is the example of left top corner of tiff 
![image](https://user-images.githubusercontent.com/22346311/34515257-8aa24cd6-f071-11e7-945a-72059f2a65f2.png)
as u can see the colours are different and also some buildings or streets were not rendered as well.

**Comment by talaj** [comment]
Ok, it is not caused by blur then.

Aren't map and data in different projections?

**Comment by mar0321** [comment]
yes map is in 3857 (mercator) cause Nik4.py dont know my srs  and i came to my ming it can be caused by wrong srs, so u think too , there might be a problem right ?

**Comment by talaj** [comment]
I think that different projection of data and map can be the culprit. It can look like trivial thing that data are automatically reprojected to the map coordinates, but it is not that simple. One should realize that bounding box of the map is a rectangle, but this bounding box is no longer a rectangle in data coordinates.

Ideally, data and map are in the same projection, but there are some Mapnik parameters that can help without need to preproject all data.

Check if `clip="true"` is not set on all the symbolizers that misbehave. Also check `buffer-size` of the map or layers. Try bigger `buffer-size`.

**Comment by mar0321** [comment]
ok thanks, i have clip false cause of elimination of cutting labels but i can try bigger buffer-size
